### PR TITLE
added Simple HTML Exporter plugin

### PR DESCRIPTION
**Pull request by pazbardanl**
### Types of changes

- [ ] Enhancement (project structure, spelling, grammar, formatting)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### A description of the changes proposed in the Pull Request

- 
-

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
   "scripts": {
     "build": "npm run clean && tsc --project tsconfig.build.json && npm run copy-files",
     "clean": "rimraf build/",
```
Comment (pazbardanl) [inline package.json:65]: hi @narekhovhannisyan I've added this line to copy the _html-export-template.html_ file to the _build_ directory on compilation (tsc), any idea why it doesn't work? :) any tip would be appreciated

Comment (narekhovhannisyan) [reply package.json:65]: seems working locally

**Comment by jmcook1186** [comment]
@pazbardanl does this still work post-refactor?

**Comment by pazbardanl** [comment]
@jmcook1186 sorry just catching up on this now. It will need to be converted to an exhaust plugin. I can do it today based on the infra established in if-exhaust-plugins but that one also needs to be reviewed

**Comment by jmcook1186** [comment]
Blocked on IF team making a decision about how exhaust/export should work.

**Comment by jmcook1186** [comment]
closing as obseleted by our new exhaust script plan. Feel free to refactor and reopen later

**Pull request closed without merging**